### ISSUE TITLE
Adding a new item to a list (e.g. shopping list) returns validation error

### ISSUE THREAD

**Issue by zHPhil93** [alexa/alexa-skills-kit-sdk-for-java#112]
## I'm submitting a...
<pre><code>[ ] Regression (a behavior that used to work and stopped working in a new release)
[X] Bug report  
[ ] Performance issue
[ ] Feature request
[ ] Documentation issue or request
[ ] Other... Please describe:
</code></pre>

For adding a new list item to Alexa's internal shopping list, i use the following code: 
   ```
 ListManagementServiceClient listClient = input.getServiceClientFactory().getListManagementService();
    listClient.createListItem(listID, CreateListItemRequest.builder()
                     .withStatus(ListItemState.ACTIVE)
                     .withValue("test value")
                     .build());

```
If i try the execution with the code above i get the following error:
`
message: 1 validation error detected: Value 'ACTIVE' at 'item.status' failed to satisfy constraint: Member must satisfy enum value set: [active, completed]`

I do not see anything wrong with my code to create a list item. Therefore i guess there is a bug in the alexa skills kit sdk for java. 

## Expected Behavior
A new item with the value "test value" and the status "ACTIVE" should be added to the alexa list (with id = listID) of the user.

## Current Behavior
Currently the skills kit returns a validation error, because the item.status with the value "ACTIVE" is not in the enum value set of [active, completed]. Therefore the list item is not created.

## Steps to Reproduce (for bugs)
```
 ListManagementServiceClient listClient = input.getServiceClientFactory().getListManagementService();
    listClient.createListItem(listID, CreateListItemRequest.builder()
                     .withStatus(ListItemState.ACTIVE)
                     .withValue("test value")
                     .build());
```

## Context

I found no other way i could add a list item to e.g. the alexa shopping list or the to-do-list with the java sdk. If there is another, i am fine with that, but would like to get some documentation on it. 

## Your Environment

* ASK Java SDK version used: 2.3.0
* Operating System and version:

## Java Info

* Java version used for development: Java 8

**Comment by breedloj** [comment]
Thanks for reporting this issue. SDK version 2.3.1 contains a fix. Feel free to let us know if you continue to experience this, or other issues.

**Comment by allusai** [comment]
Hello there!

I think someone had expressed a similar issue in this discussion forum: https://forums.aws.amazon.com/thread.jspa?threadID=175775.

From what I understand, the problem isn't how you are adding an item to a list, but rather with how you set up the DynamoDB table. When you write the AttributeDefinitions for an Attribute, you only should give an AttributeDefinition to primary key Attributes. So somewhere in your code you might be assigning an attribute definition to a non-primary key attribute.

Let me know if this works!
Sai

**Comment by zHPhil93** [comment]
Hi, 

thanks for your feedbakc. The bug is fixed with version 2.3.1, as @breedloj stated. 
The forum link provided by @allusai is a completely different topic. I am not using any DynamoDB table, i just want to add an item to the shopping list of the customer. 

Thanks for your quick help!
Philipp
